src/config.xml

```diff
@@ -4151,9 +4151,21 @@ to be found in the default search path.
 <![CDATA[
  When using PlantUML, the \c PLANTUML_JAR_PATH tag should be used to specify the path where
  java can find the \c plantuml.jar file or to the filename of \c jar file to be used.
+ See \c PLANTUML_BIN_PATH if you use a native image.
  If left blank, it is assumed PlantUML is not used or
  called during a preprocessing step. Doxygen will generate a warning when it encounters a
  \ref cmdstartuml "\\startuml" command in this case and will not generate output for the diagram.
+]]>
+      </docs>
+    </option>
+    <option type='string' id='PLANTUML_BIN_PATH' format='file' defval=''>
```

Comment:
Instead of a yet another option, which in this case seems to be mutual exclusive to `PLANTUML_JAR_PATH`, why not allow `PLANTUML_JAR_PATH` to optionally point to a non-jar file and then treat that as if `PLANTUML_BIN_PATH` were used. Would that work?

Reply:
Hey i think that would absolutely work these where kind of the 2 options i saw to implement this and since the discussion in the issue did not uncover a preference i just went for one
![image](https://github.com/user-attachments/assets/9ecaf73f-b91a-4a4f-a85a-07c461200ed5)

The downside of reusing the `PLANTUML_JAR_PATH` in my opinion are that this will be more convoluted to configure and maybe a bit harder to figure out. But as you said an other option as implemented here is also not without its downsides.

So if you think we should go for a reuse/extend of the option `PLANTUML_JAR_PATH` i can absolutely update the MR to do that just let me know how you would like to see this work :)

Reply:
updated & rebased to master to reuse the existing config flag instead of introducing a new flag @doxygen @albert-github